com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl

```diff
@@ -128,18 +129,23 @@ TEXTURE2D_X(_CustomColorTexture);
 // Main lightmap
 TEXTURE2D(unity_Lightmap);
 SAMPLER(samplerunity_Lightmap);
+TEXTURE2D_ARRAY(unity_Lightmaps);
+SAMPLER(samplerunity_Lightmaps);
+
 // Dual or directional lightmap (always used with unity_Lightmap, so can share sampler)
 TEXTURE2D(unity_LightmapInd);
+TEXTURE2D_ARRAY(unity_LightmapsInd);
 
 // Dynamic GI lightmap
 TEXTURE2D(unity_DynamicLightmap);
 SAMPLER(samplerunity_DynamicLightmap);
 
 TEXTURE2D(unity_DynamicDirectionality);
 
-// We can have shadowMask only if we have lightmap, so no sampler
 TEXTURE2D(unity_ShadowMask);
 SAMPLER(samplerunity_ShadowMask);
+TEXTURE2D_ARRAY(unity_ShadowMasks);
+SAMPLER(samplerunity_ShadowMasks_linear_clamp);
```

Comment:
guess we can have a single SAMPLER(samplerunity_ShadowMasks_linear_clamp); and remove SAMPLER(samplerunity_ShadowMask); ?  is it a different sampler?

Reply:
This is wrong on my side this should be samplerunity_ShadowMasks to match the array name. I thought i fixed this yesterday but missed it for for hdrp

Reply:
Fixed locally.